controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -185,7 +186,7 @@ func (r *StorageConsumerReconciler) reconcilePhases() (reconcile.Result, error)
 
 		if r.verifyCephResourcesDoNotExist() {
 			r.Log.Info("Removing finalizer from StorageConsumer.", "StorageConsumer", klog.KRef(r.storageConsumer.Namespace, r.storageConsumer.Name))
-			r.storageConsumer.ObjectMeta.Finalizers = remove(r.storageConsumer.ObjectMeta.Finalizers, storageConsumerFinalizer)
+			r.storageConsumer.ObjectMeta.Finalizers = Remove(r.storageConsumer.ObjectMeta.Finalizers, storageConsumerFinalizer)
 			if err := r.Client.Update(r.ctx, r.storageConsumer); err != nil {
 				r.Log.Error(err, "Failed to remove finalizer from StorageConsumer", "StorageConsumer", klog.KRef(r.storageConsumer.Namespace, r.storageConsumer.Name))
 				return reconcile.Result{}, err
```

Comment:
There should be no finalizers on the storageclassclaim in the first place. Adding a finalizer will only work against our  delete process - see same comment on the claim controller